modules/conflict_resolver/php/NDB_Menu_Filter_resolved_conflicts.class.inc

```diff
@@ -57,31 +57,33 @@ class NDB_Menu_Filter_Resolved_Conflicts extends NDB_Menu_Filter
         }
 
         $this->columns = array(
-                          'conflicts_resolved.TableName',
-                          'session.CandID',
-                          'candidate.PSCID',
-                          'session.Visit_label',
-                          'conflicts_resolved.ResolutionTimestamp',
+                          'conflicts_resolved.TableName as Instrument',
+                          'session.CandID as CandID',
+                          'candidate.PSCID as PSCID',
+                          'session.Visit_label as Visit_Label',
+                          'conflicts_resolved.ResolutionTimestamp as Timestamp',
                          );
 
         if ($useProjects) {
-            $this->columns[] = 'candidate.ProjectID as Project';
+            $this->columns[] = 'Project.Name as Project';
         }
 
         $this->columns = array_merge(
             $this->columns,
             array(
-             'conflicts_resolved.FieldName',
-             'conflicts_resolved.NewValue',
-             'conflicts_resolved.OldValue1',
-             'conflicts_resolved.OldValue2',
-             'session.CenterID',
+             'conflicts_resolved.FieldName as Question',
+             'conflicts_resolved.OldValue1 as CorrectAnswer',
+             'conflicts_resolved.NewValue  as New_Value',
+             'conflicts_resolved.OldValue2 as OldValue2',
+             'session.CenterID as CenterID',
```

Comment:
The aliases and naming are indeed somewhat confusing, but essentially `NewValue` tracks the selection user made in unresolved conflicts (either 1 or 2). So if user selected first option, the table would check 
`if (newvalue == 1) correctAnswer = OldValue1`
`if (newvalue == 2) correctAnswer = OldValue2`
`display correctAnswer`